plugins/src/main/java/com/google/fhir/gateway/plugin/ListAccessChecker.java

```diff
@@ -171,8 +171,9 @@ public AccessDecision checkAccess(RequestDetailsReader requestDetails) {
           return processPut(requestDetails);
         case PATCH:
           return processPatch(requestDetails);
+        case DELETE:
+          return processDelete(requestDetails);
```

Comment:
Sorry, one thing that I remembered after sending the previous review is handling DELETE in a Bundles which we should address as part of #43 . This also applies to #72. If you want to have a separate PR for that, it is fine, but please do so before closing #43.